test/test_jit.py

```diff
@@ -15166,14 +15165,6 @@ def parameter_script(x: torch.nn.Parameter):
     'test_nn_max_pool1d_with_indices',
 }
 
-def check_alias_annotation(method_name, args, kwargs):
-    formals, tensors, actuals = get_script_args(args)
-    call = get_call(method_name, 'method', actuals, kwargs)
-    script = script_template.format(', '.join(formals), call)
-    CU = torch.jit.CompilationUnit(script)
-    torch._C._jit_check_alias_annotation(CU.the_method.graph, tuple(tensors), method_name)
```

Comment:
this is no longer being called - it's not checking op normalization of aliasing, it's checking alias annotations of schemas

Reply:
Aha! I wanted to check on that. I'll revert this change.